docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -1,9 +1,246 @@
-= Get Started with Hazelcast CLC
+= Get Started With the Hazelcast CLC
+:description: In this tutorial, you'll learn the basics of how to configure, start up, and connect the Hazelcast Command Line Client (CLC) to a {hazelcast-cloud} cluster. You'll also see how to switch between clusters using the CLC. Finally, you'll perform some basic operations on a cluster from the command line and then by running a script to automate the same actions.
 
-You can use Hazelcast CLC to manage {hazelcast-cloud} clusters or self-managed installations of Hazelcast Platform.
+{description}
 
-Choose the Hazelcast product that you want to use:
+== Before You Begin
 
-- {hazelcast-cloud}
-- Hazelcast Platform
+You need the following:
+
+- xref:install-clc.adoc[Hazelcast CLC installed] on your local machine
+- Two xref:cloud:ROOT:create-serverless-cluster.adoc[Serverless clusters]. For this exercise, we'll use one development and one production cluster, to show how you might use the CLC for basic administration and deployment tasks.
+- Connection credentials and security keys for both clusters.
+
+TIP: You can find all connection credentials, such as the cluster name and discovery token, and security keys in the {hazelcast-cloud} console. For details, see xref:cloud:ROOT:connect-to-cluster.adoc[]. 
+
+[[step-1-dev-config]]
+== Step 1. Configure the CLC for a Development Cluster
+
+Before you can connect the CLC to your development cluster, you need to add the cluster's security keys and connection details to the default configuration file.
+
+. Save the following security keys to a location on your local machine that's easy to find.
+
+** `ca.pem`
+** `cert.pem`
+** `key.pem`
++
+For example: `$HOME/.local/share/*dev*`.
+
+. Go to the directory `$HOME/.local/share/hz-cli` and open `config.yaml`. This is the default configuration file for the CLC.
+
+. Add the following configuration details for your development cluster:
++
+[source,yaml]
+----
+ssl:
+  enabled: true
+  capath: "/<PATH>/ca.pem"
+  certpath: "/<PATH>/cert.pem"
+  keypath: "/<PATH>/key.pem"
+  keypassword: "<DEV_KEYSTORE_PASSWORD>"
+hazelcast:
+  cluster:
+    name: "<DEV_CLUSTER_NAME>"
+    cloud:
+      token: "<DEV_DISCOVERY_TOKEN>"
+      enabled: true
+----
++
+NOTE: Different parameter values are required when connecting to clusters on Hazelcast Platform. See xref:configuration.adoc[] for full details.
+. Save your changes.
+
+[[step-2-dev-connect]]
+== Step 2. Connect the CLC to the Development Cluster
+You're ready to connect the CLC to your development cluster. Make sure that the cluster is running before you start.
+
+. From the command line, change into `$HOME/.local/share/hz-cli`.
+. Execute the following command:
++
+```bash
+hzc
+```
++
+The CLC connects to the development cluster and the command prompt displays the cluster details. For example: `hzc hazelcast-cloud@pr-3145>`.
```

Comment:
```suggestion
The Hazelcast CLC connects to the development cluster and the command prompt displays the cluster details. For example: `hzc hazelcast-cloud@pr-3145>`.
```

Reply:
Check if the output will continue to include `hazelcast-cloud` even when Serverless beta is launched.

Reply:
Query is with client team.